pymongoexplain/__main__.py

```diff
@@ -13,48 +13,35 @@
 # limitations under the License.
 
 
-from pymongo import monitoring, MongoClient
 from pymongo.collection import Collection
+from .explainable_collection import ExplainCollection
+
 import sys
-from bson.son import SON
+import logging
```

Comment:
Can you add a brief module-level docstring?

Reply:
Done.